doc/modules/ROOT/pages/rel-embedding-models.adoc

```diff
@@ -53,30 +53,41 @@ Since the prediction parts of the computations in these methods are the same, th
 | Name               | Type                       |
 | source_node_filter | Union[str, int, list[int]] | The specification of source nodes to consider. Either a node label, a node ID, or a list of node IDs
 | target_node_filter | Union[str, int, list[int]] | The specification of source nodes to consider. Either a node label, a node ID, or a list of node IDs
+| relationshipTypes  | str or list[str]           | Relationship types for which the algorithm will be executed
 | top_k              | int                        | How many relationships to produce for each source node. The target nodes with the highest `top_k` scores will be kept for each source node
 | general_config     | **dict[str, Any]           | General GDS algorithm configuration as optional keyword parameters
 |===
 
-In particular, the general algorithm configuration parameters supported as keyword parameters for this algorithm are `concurrency`, `nodeLabels`, `relationshipTypes`, `jobId` and `logProgress`.
+In particular, the general algorithm configuration parameters supported as keyword parameters for this algorithm are `concurrency`, `jobId` and `logProgress`.
 You can read more about them https://neo4j.com/docs/graph-data-science/current/common-usage/running-algos/#algorithms-syntax-configuration-parameters[here] in the GDS manual.
 
 Let us now outline the differences between these prediction methods.
 
 
 === Streaming predicted relationships
 
-The `SimpleRelEmbeddingModel.predict_stream` method returns a `pandas.DataFrame` which contains three columns: `sourceNodeId`, `targetNodeId` and `score`.
+The `predict_stream` method returns a `pandas.DataFrame` which contains three columns: `sourceNodeId`, `targetNodeId` and `score`.
 These refer to the source node ID, the target node ID, and the score from running the KGE model scoring function on the node pair and relationship type, respectively.
 
 There are no extra parameters to this method other than the ones outlined xref:rel-embedding-models.adoc#rel-emb-predict-shared-params[above].
 
 
 === Mutating graph projection with predicted relationships
 
-The `SimpleRelEmbeddingModel.predict_mutate` method adds the predicted relationships to the graph projection under a new type specified via the `mutate_relationship_type` parameter.
+The `predict_mutate` method adds the predicted relationships to the graph projection under a new type specified via the `mutate_relationship_type` parameter.
 Each such relationship will have a property, specified via the `mutateProperty` parameter, representing the output from running the KGE model scoring function on the node pair and relationship type.
 The method returns a `pandas.Series` with metadata about the computation:
 
+In addition to the shared parameters outlined xref:rel-embedding-models.adoc#rel-emb-predict-shared-params[above], this method takes two more positional parameters, after the `top_k` parameter, in order:
```

Comment:
looks risky to mix inheriting parameters and using positional parameters. creates a dependency in the documentation and probably also in the code. if the shared parameters add another parameter then the "after the `top_k`" parameter needs to be updated. in gds mutateProperty is not positional, so why not keep it like a keyword argument here?

Reply:
This phrase was just moved upper. I agree with you, I think it has to be keyword argument.